src/arrayReverse.test.js

```diff
@@ -16,5 +16,30 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
-  // write more tests here
+  it(`should return an array equal to the length `
+  + `of the original array`, () => {
+    const words = ['hello', 'world', 'test'];
+    const reversedWords = arrayReverse(words);
+
+    expect(reversedWords.length).toBe(words.length);
+  });
+
+  it(`should return an array with reverse string`, () => {
+    const result = arrayReverse(['Hell0']);
+
+    expect(result).toEqual(['0lleH']);
+  });
+
+  it(`should return the same length of first string`
+  + `in massive after reverse`, () => {
+    const result = arrayReverse(['Mate', 'Academy']);
+
+    expect(result[0].length).toEqual(4);
+  });
+
+  it(`should works with symbols and numbers`, () => {
```

Comment:
The test case description is not clear and could be misunderstood. It would be better to write 'should correctly reverse strings containing symbols and numbers'.